### PR TITLE
Implement home speakers section

### PR DESCRIPTION
**Pull request by BKinya**
# Scope

_Please make sure to read https://github.com/droidconKE/droidconKE2022Android/docs/CONTRIBUTING.md
and check that you understand and have followed it as best as possible,
 Explain what your feature does in a short paragraph.

Implemented the speakers' section on the home screen. 

_ please check the below boxes
- [x] I have followed the coding conventions
- [x] I have added/updated necessary tests
- [x] I have tested the changes added on a physical device
- [x] I have run `./codeAnalysis.sh` to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Please add a screenshot (if necessary)
speakers section

### CODE REVIEW

File: presentation/src/main/java/com/android254/presentation/speakers/view/SpeakersScreen.kt [hunk 1]
```diff
@@ -56,7 +59,7 @@ fun SpeakersScreen(speakersViewModel: SpeakersViewModel = hiltViewModel()) {
                 },
                 navigationIcon = {
                     IconButton(
-                        onClick = { },
+                        onClick = navigateToHomeScreen,
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/speakers/view/SpeakersScreen.kt:62]: we can use navigate back function available to the navController

File: presentation/src/main/java/com/android254/presentation/home/components/HomeSpeakerComponent.kt [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            modifier = Modifier.constrainAs(speakerName) {
+                top.linkTo(headShot.bottom, 10.dp)
+                start.linkTo(parent.start)
+                end.linkTo(parent.end)
+            },
+            style = TextStyle(
+                color = colorResource(id = R.color.dark),
+                fontSize = 12.sp,
+                fontFamily = FontFamily(
+                    Font(R.font.montserrat_medium)
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/home/components/HomeSpeakerComponent.kt:91]: Use the font resource available in Chai. Check home header for inspiration

**Review by chepsi — APPROVED** [review]
Looks good. Minor improvements suggested.

**Review by misshannah — APPROVED** [review]
Awesome!

**Review by michaelbukachi — APPROVED** [review]
LG

**Pull request merged**